tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-ortmodule-test-clear-cache-pipeline.yml

```diff
@@ -0,0 +1,25 @@
+trigger: none
+
+jobs:
+- job: Onnxruntime_Linux_GPU_ORTModule_Clear_Cache
+
+  timeoutInMinutes: 120
+  pool: 'Linux-Single-GPU-V100'
+
+  steps:
+  - checkout: self
+    clean: true
+    submodules: recursive
+
+  - bash: tools/ci_build/github/linux/docker/scripts/training/azure_scale_set_vm_mount_test_data.sh -p $(orttrainingtestdata-storage-key) -s "//orttrainingtestdata.file.core.windows.net/hf-models-cache" -d "/hf_models_cache"
+    displayName: 'Mount hf-models-cache'
+    condition: succeededOrFailed()
+
+  - bash: rm /hf_models_cache/huggingface/transformers/*
```

Comment:
is $data_destination always get removed in the first step in azure_scale_set_vm_mount_test_data.sh?

Reply:
I think the intention here is slightly different.
In ```azure_scale_set_vm_mount_test_data.sh```, the first step is to unmount the volume (if it was already mounted) and then delete the directory.
In this case, the intention is to delete the contents of the mounted volume so that all the contents that were stored previously in the storage container are deleted.
Then, on the next pull request pipeline run, these contents will be redownloaded and populated on the storage container.
That is the intention behind this ```rm``` here.